mars/services/session/api.py

```diff
@@ -159,5 +159,5 @@ async def create_session(self, session_id: str) -> str:
     async def delete_session(self, session_id: str):
         return await self._call_method({}, 'delete_session', session_id)
 
-    async def last_idle_time(self, session_id: Union[str, None] = None) -> Union[float, None]:
+    async def get_last_idle_time(self, session_id: Union[str, None] = None) -> Union[float, None]:
         return await self._call_method({}, 'last_idle_time', session_id)
```

Comment:
Should be `return await self._call_method({}, 'get_last_idle_time', session_id)`? It's not necessary to change the API name, the name `last_idle_time` is clear enough.

Reply:
@fyrestone We recommend all method names to be verb phrases.